ebsco_adapter/ebsco_adapter/src/main.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```

Comment:
I assume this file needs on review, as it just exists so we have something for the skeleton to run.